### PR TITLE
Does my coding partner follow JS best practices?

### PR DESCRIPTION
**Pull request by brighton0102**
### In this milestone I :

- Create a new empty repo.
- Copy all the code from the "To Do list" project and commit it as one big chunk of code in a feature branch.

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+}
+
+.container {
+  width: 100%;
```
Comment (efmomoh) [inline src/styles/main.css]: In following the best CSS practices, please make sure that there is no redundancy or duplication of CSS key-value pairs in our work. Let's make sure that we are organized in a way that ensures our code readability and maintainability. Example: if our CSS has **.container  set to wdith: 100% and #wrapper is set to width: 100%**, make sure that we have a single selector assigned to all this elements and give them a width of 100%. This ensures code readability and maintainability.

File: src/modules/markAsComplete.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const markAsComplete = () => {
+  const markComplete = document.querySelectorAll('.check-box');
+  markComplete.forEach((element, index) => {
+    element.addEventListener('input', (event) => {
+      const items = JSON.parse(localStorage.getItem('toDoList')) || [];
+      items[index].completed = event.target.checked;
+      localStorage.setItem('toDoList', JSON.stringify(items));
+      // eslint-disable-next-line no-restricted-globals
+      location.reload(true);
```
Comment (efmomoh) [inline src/modules/markAsComplete.js:9]: Please ensure that we do not disable eslint in our code base, we can fix it by placing the disable statement to the webpack bundlers and config files. Kindly remove eslint disable comments from the various files.

Comment (brighton0102) [reply src/modules/markAsComplete.js:9]: Thanks for the review it's really a good practice to maintain eslint checks in my codebase and not disable them directly within the code. 💯

**Review by efmomoh — CHANGES_REQUESTED** [review]
### Hi Team @brighton0102,

Good job so far 👍 

Good implementation of some project specifications

Well Implemented files structure ✅
README.md ✅
Follow best modular practices ✅

### HOWEVER,  CHANGES  ARE REQUIRED ♻️ 

- [ ] [REQUIRED] Kindly ensure that linters check

**KINDLY CHECK THE COMMENT SECTION ABOVE TO IMPLEMENT THE REQUIRED CHANGES, WHEN DONE PLEASE RESUBMIT FOR A RE-REVIEW.**


Have any questions or doubts? Feel free to drop me a tread in [Slack](https://app.slack.com/client/T47CT8XPG/C05PHV5KYS2/rimeto_profile/U04KV0FNRCM)

File: src/modules/markAsComplete.js (hunk 2 above)
Comment (efmomoh) [reply src/modules/markAsComplete.js:9]: > Thanks for the review it's really a good practice to maintain eslint checks in my codebase and not disable them directly within the code. 💯

You are welcome. Feel free to request a re-review when you are done implementing the changes 👌

Comment (brighton0102) [reply src/modules/markAsComplete.js:9]: Hi @efmomoh 

For me to use eslint-disable-next-line comment is really necessary for my project to run smoothly I've tried other ways hope you can understand my dilemma 

below is a tip from my previous project

TIP: If you separated the code in multiple files and you have the following issues with the linter: (i) complaining about having multiple classes at the same file, or (ii) unused variables, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */.

Thank you in advance

Comment (efmomoh) [reply src/modules/markAsComplete.js:9]: @brighton0102, Based on the specifications, a Linters check is required. That means if our project linters can not check that means there should be a major issue with our syntaxes and structure of files. However, you may esLint disable the particular line of code that is making complaints and push your changes thereby enabling linters to run on GitHub workflows. When you are done, ask for a re-review.

**Comment by brighton0102** [comment]
Hi @efmomoh 

Now all the linter checks.

**Review by efmomoh — APPROVED** [review]
### **Hurrah STATUS: APPROVED 💪💪💪   👍👏


Hi Team @brighton0102 CONGRATULATIONS 🏆🏆

Your project is complete....there is nothing else to say other than it's time to merge 👍

Linters passed and followed best practices ✅

**Pull request merged**